data/core/step1_get_started.md

```diff
@@ -180,6 +180,32 @@
     - utter_no_guide_for_switch
     - utter_anything_else
 
+
+## just switch
+* switch{"current_api":"tensorflow"}
+    - slot{"current_api":"__other__"}
```

Comment:
Does `__other__` have a special meaning, compared to `other` ?

Reply:
yes, it's the reserved key for categorical slot values that aren't specified in the domain